pkg/sandbox/worker.go

```diff
@@ -150,6 +150,14 @@ func (w *Worker) UpdateChunkState() {
 }
 
 func (w *Worker) UpdateChunkTemp() {
+
+	directions := [][]int{
+		{0, -1}, // bottom
+		{0, 1},  // up
```

Comment:
```suggestion
		{0, -1}, // up
		{0, 1},  // bottom
```

The top left corner is 0,0